src/DTO/DefaultStrategy.php

```diff
@@ -8,12 +8,14 @@ final class DefaultStrategy implements Strategy
      * @param array<string,string> $parameters
      * @param array<Constraint>    $constraints
      * @param array<Segment>       $segments
+     * @param array<Variant>       $variants
      */
     public function __construct(
         private readonly string $name,
         private readonly array $parameters = [],
         private readonly array $constraints = [],
         private readonly array $segments = [],
+        private readonly array $variants = [],
```

Comment:
This is a backwards compatibility break, new parameters should be added as last.